app/models/UserAnswers.scala

```diff
@@ -28,7 +28,8 @@ final case class UserAnswers(
   lastUpdated: Instant,
   id: UUID,
   status: SubmissionState,
-  departureId: Option[String] = None
+  departureId: Option[String] = None,
+  isTransitional: Option[Boolean] = Some(true)
```

Comment:
We could potentially have this as a non-optional. Reads would just default to true if it is undefined.